### DISCUSSION TITLE
Will pino 7.x make it into a Fastify 3.x release, or do we wait for the next major?

### DISCUSSION THREAD

**Discussion by saumets** [fastify/fastify#3455] [Q&A] [answered]
We had migrated to using `pino` 7 with our Express server a short while back, but now we are also migrating away from Express and adopting Fastify. Just noticed that the existing Fastify 3.x major still uses `pino` 6 while the `next` branch has version 7. 

After adopting the new transport system in `pino` 7 we don't really want to downgrade, and re-create our necessary streams again. So we're just wondering if `pino` 7 will see light on Fastify 3.x or are we waiting for Fastify 4?

**Comment by delvedor** [comment, marked as answer]
Pino v7 will be the default only in Fastify v4.
You should be able to use Pino v7 in Fastify v3 by configuring a [custom logger](https://www.fastify.io/docs/latest/Server/#logger) :)
